src/Stryker.Core/Stryker.Core.UnitTest/Mutants/CsharpMutantOrchestratorTests.cs

```diff
@@ -13,67 +13,75 @@ public class CsharpMutantOrchestratorTests : MutantOrchestratorTestsBase
     [Fact]
     public void ShouldNotMutateEmptyInterfaces()
     {
-        var source = @"using System;
-using System.Collections.Generic;
-using System.Text;
-namespace StrykerNet.UnitTest.Mutants.TestResources
-{
-    interface TestClass
-    {
-        int A { get; set; }
-        int B { get; set; }
-        void MethodA();
-    }
-}";
-
-        var expected = @"using System;
-using System.Collections.Generic;
-using System.Text;
-namespace StrykerNet.UnitTest.Mutants.TestResources
-{
-    interface TestClass
-    {
-        int A { get; set; }
-        int B { get; set; }
-        void MethodA();
-    }
-}";
-        ShouldMutateSourceToExpected(source, expected);
+        const string Source = """
```

Comment:
Any specific reason fro switching to raw strings?
I do not feel this has significant benefits here

Reply:
I see what you mean, however, I think this improves readability. Also, Microsoft [recommends](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/strings/#raw-string-literals) to use them when dealing with pre-formatted text. I think it's more suitable for source code snippets as those used in this test file.

Reply:
I u de it comes from a good place. But this kind of change drags a huge, invisible cost: PRs

Any PR referencing the previous version will be painful to merge. 
I am not sure the benefits outweigh this drawback. 
Been there, done that.
We will need to check with the core team what they feel about this.